### ISSUE TITLE
Scalable keyboard accessibility for Language Dropdown in User Setttings Dialog

### ISSUE THREAD

**Issue by UlianaMunich** [SAP/luigi#2565]
[DXP Issue-origin](https://github.tools.sap/dxp/backlog/issues/391)

- When the dropdown is open, it should be able to navigate through the list with DOWN and UP. Not with TAB, because this should take focus to the "Save" button.
- After choosing a value from the select list, the focus should remain on the select.
- When opening the dropdown list, the focus should be already set on the selected list item.

UI references from UI5:https://sapui5.hana.ondemand.com/#/entity/sap.m.Select/sample/sap.m.sample.Select

Keep in mind other accessibility functionality from this ticket #2453
